presto-main/src/test/java/io/prestosql/type/TestTimeWithTimeZoneBase.java

```diff
@@ -198,7 +200,12 @@ public void testBetween()
     }
 
     @Test
-    public abstract void testCastToTime();
+    public void testCastToTime()
+    {
+        assertFunction("cast(TIME '03:04:05.321 +07:09' as time)",
```

Comment:
add a case with `as time(1)` and `as time(9)` as well

Reply:
There are a bunch of tests in io.prestosql.operator.scalar.timestamptz.TestTimestampWithTimeZone.testCastToTime